lib/index.js

```diff
@@ -64,16 +64,16 @@ rpc.on('session clear req', () => {
   store_.dispatch(sessionActions.clearActiveSession());
 });
 
-rpc.on('termgroup add req', () => {
-  store_.dispatch(termGroupActions.requestTermGroup());
+rpc.on('termgroup add req', (options) => {
```

Comment:
Please remove the parenthesis around `options` – it's not needed for a single function argument